__tests__/sendTypingIndicator.js

```diff
@@ -20,21 +23,52 @@ test('Send typing indicator', async () => {
   await input.sendKeys('ABC');
 
   // Typing indicator takes longer to come back
-  await driver.wait(minNumActivitiesShown(3), 5000);
+  await driver.wait(receivedTypingActivity(), timeouts.directLine);
```

Comment:
Matching other naming in `/__tests__/setup/conditions/`.

```suggestion
  await driver.wait(typingActivityReceived(), timeouts.directLine);
```